docs/content/en/docs/concepts/tracing-policy/example.md

```diff
@@ -0,0 +1,144 @@
+---
+title: "Example"
+icon: "tutorials"
+weight: 1
+description: "Learn the basics of Tracing Policy via an example"
+---
+
+To discover `TracingPolicy`, let's understand via an example that will be
+explained, part by part, in this document:
+
+```yaml
+apiVersion: cilium.io/v1alpha1
+kind: TracingPolicy
+metadata:
+  name: "fd-install"
+spec:
+  kprobes:
+  - call: "fd_install"
+    syscall: false
+    args:
+    - index: 0
+      type: "int"
+    - index: 1
+      type: "file"
+    selectors:
+    - matchArgs:
+      - index: 1
+        operator: "Equal"
+        values:
+        - "/etc/passwd"
+        - "/etc/shadow"
+      matchAction:
+      - action: Sigkill
+```
+
+## Required fields
+
+```yaml
+apiVersion: cilium.io/v1alpha1
+kind: TracingPolicy
+metadata:
+  name: "fd-install"
+```
+
+The first part follows a common pattern among all Cilium Policies or more
+widely [Kubernetes object](https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/).
+It first declares the Kubernetes API used, then the kind of Kubernetes object
+it is in this API and an arbitrary name for the object, that has to comply with
+[Kubernetes naming convention](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/).
+
+## Hook point
+
+```yaml
+spec:
+  kprobes:
+  - call: "fd_install"
+    syscall: false
+    args:
+    - index: 0
+      type: "int"
+    - index: 1
+      type: "file"
+```
+
+The beginning of the specification describe the hook point to use, here we
+are using a kprobe, hooking on the kernel function `fd_install`. We indicate
+that it's not a syscall (which is by the way unecessary since false is the
+default). Then we specify the argument of the specified function symbol to be
+able to extract them, and optionally perform filtering.
```

Comment:
```suggestion
The beginning of the specification describes the hook point to use. In this case, it is 
a kprobe, hooking on the kernel function `fd_install`. We indicate
that it's not a syscall (can be omitted, since this is the default). We also specify arguments and their type, which will  instruct Tetragon to extract them and include them in any resulting events, as well enable filtering based on their values in subsequent selectors.
```

Reply:
Also just noticed that syscall is true by default: https://github.com/cilium/tetragon/blob/main/pkg/k8s/apis/cilium.io/v1alpha1/types.go#L118-L120

Reply:
I removed that sentence.